README.md

```diff
@@ -1,25 +1,48 @@
-# A platform for the exploration and evaluation of Hanabi AIs
+# Intentional AI (IntAI) for Hanabi
 
-## Synopsis
-
-This repository contains an implementation of the card game [Hanabi](https://boardgamegeek.com/boardgame/98778/hanabi) with several different AIs. It can be used to see how current AIs perform, as well as to develop new AIs, including testing them with human cooperators.
+This repository contains an implementation of the card game [Hanabi](https://boardgamegeek.com/boardgame/98778/hanabi)
+with several different AIs, most notably Intentional AI created by
+Eger et al. (2017).
 
 ## Usage
 
-Our implementation has two modes of operation: A graphical one, running inside a web-browser, and a command-line option. To use the graphical interface, run:
+### Prerequisites
+
+By default, our simulator uses the Python binding of [HanaSim](https://github.com/UofT-Hanabi-Lab/HanaSim/tree/pyhanabi-xp)
+to handle the game state.
+You must build the binding and install it into our simulator, following the
+steps below:
+
+1. From the [pyhanabi-xp](https://github.com/UofT-Hanabi-Lab/HanaSim/tree/pyhanabi-xp)
+    branch of HanaSim, follow the instructions in `Instruction.md` to build the
+    binding binary.
+   1. You must build the binding using the same version of Python that you'll
+       use to run IntAI (e.g. Python 3.12).
+2. Copy the binary into your IntAI env with the name `hana_sim.so`
+    (e.g. `.venv/lib/python3.12/site-packages/hana_sim.so`).
+
+All Python dependencies are specified in the `pyproject.toml` at the repo root.
+

```

Comment:
I think it would be better if we add a section about setting up the environment after Prerequisites section.
Like mention explicitly what to do on different systems. On Linux, after the placing` hana_sin.so` to the correct place, run `uv sync`, then run `source .venv/bin/activate` to activate the environment, then import hana_sim package by `python -c 'import hana_sim`  `ldd .venv/lib/python3.12/site-packages/hana_sim.so`(I'm not 100% sure whether this is the correct command line)